examples/stable_diffusion_xl/gm/modules/attention.py

```diff
@@ -20,10 +22,14 @@
     FLASH_IS_AVAILABLE = True
     print("flash attention is available.")
 except ImportError:
+    FlashAttention = None
```

Comment:
`FlashAttention = None` seems not necessary

Reply:
it's necessary because `stable_diffusion_xl/gm/modules/diffusionmodules/model.py` imports `FlashAttention` without verifying its availability (e.g. not available on GPU and thus, execution crashes). So, need to either fix importing or set `FlashAttention` to `None`. I went with the easier fix.